docs/version-history/lts-versions-23-0.rst

```diff
@@ -34,6 +34,11 @@ New Features:
 
 Bugs Fixed:
 
+- Changed the value of the HoldSubCode when a job goes on hold due to a file
+  transfer plugin error.  Now the HoldSubCode contains the exit code of the
+  plugin process.
+  :jira:`2373`
```

Comment:
Something like

```suggestion
- Fixed bug where :attr:`HoldReasonSubcode` was not its documented value for file transfer plugin errors.
  :jira:`2373`
```

would be less confusing, I think.

Reply:
Note we don't have the :attr: tags in the stable branch of the docs.